src/Core/Worker.php

```diff
@@ -626,34 +626,14 @@ public static function tooMuchWorkers()
 			$maxsysload = intval(Config::get("system", "maxloadavg", 50));
 
 			if (Config::get('system', 'worker_linear_load', false)) {
```

Comment:
Please put the exponent into a config variable - with "3" as default. Additionally please leave the comments in some form in the code so that people will know the difference.